src/main/java/org/broadinstitute/hellbender/tools/spark/sv/utils/StructuralVariantContext.java

```diff
@@ -127,7 +140,8 @@ public int getEnd() {
      * @param pipelineOptions pipeline options.
      * @return never {@code null}.
      */
-    public Haplotype composeHaplotypeBasedOnReference(final int index, final int paddingSize, final ReferenceMultiSource reference, final PipelineOptions pipelineOptions)  {
+    public Haplotype composeHaplotypeBasedOnReference(final int index, final int paddingSize, final ReferenceMultiSource reference,
+                                                      final PipelineOptions pipelineOptions)  {
```

Comment:
My last comment about `PipelineOptions` is probably overwelmed by other changes, but here's the summary:
My suggestion about adding PipelineOptions are wrong, and please remove them. We need only null for them practically.